doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -128,36 +128,40 @@ Update ACRN Hypervisor Image
    .. code-block:: none
 
       $ sudo lspci -vv
-      00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
-              Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
-      00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
-              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
-      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
-              Subsystem: Intel Corporation Ethernet Connection I219-LM
+      00:14.0 USB controller: Intel Corporation Device 9ded (rev 30) (prog-if 30 [XHCI])
+              Subsystem: Intel Corporation Device 7270
+      00:17.0 SATA controller: Intel Corporation Device 9dd3 (rev 30) (prog-if 01 [AHCI 1.0])
+              Subsystem: Intel Corporation Device 7270
+      02:00.0 Non-Volatile memory controller: Intel Corporation Device f1a8 (rev 03) (prog-if 02 [NVM Express])
+              Subsystem: Intel Corporation Device 390d
+      03:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
+              Subsystem: Intel Corporation I210 Gigabit Network Connection
+      04:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
+              Subsystem: Intel Corporation I210 Gigabit Network Connection
 
    .. note::
-      Verify the PCI devices BDF defined in the
-      ``hypervisor/arch/x86/configs/whl-ipc-i5/pci_devices.h``
+      Verify the ``pci_devs`` defined for VM0 and VM1 in the
```

Comment:
I think we should also add a note here to indicate that if changes are needed, a copy of the file should be made somewhere and subsequently used when building ACRN (via the `SCENARIO=/path/to/modified/scenario.xml`

Reply:
Good idea! Please add the note in PR directly

Reply:
I have pushed a new commit to your branch to add this note. It is now showing directly in this PR.